### PR TITLE
Set global configuration for JDK17

### PR DESCRIPTION
**Pull request by mcarlett**

### CODE REVIEW

**Comment by mcarlett** [comment]
@osmman @tveskrna @Croway I have removed `fuse` tests from the test workflow because Fuse 6 is not working with JDK17, we can restore it once Fuse 7.12 will be available

File: containers/eap/src/main/java/org/jboss/qa/jcontainer/eap/EapConfiguration.java [hunk 1]
```diff
@@ -39,10 +39,7 @@ public Builder() {
 			super();
 			xms("1303m");
 			xmx("1303m");
-			//-XX:MaxPermSize has been removed from jdk17
```
Comment (osmman) [inline containers/eap/src/main/java/org/jboss/qa/jcontainer/eap/EapConfiguration.java:42]: why did you revert this change

Comment (mcarlett) [reply containers/eap/src/main/java/org/jboss/qa/jcontainer/eap/EapConfiguration.java:42]: moved to common code https://github.com/jboss-qa/jcontainer-manager/blob/jdk17-move-to-java-config/core/src/main/java/org/jboss/qa/jcontainer/JavaConfiguration.java#L118-L138
